i18n/locales/en.ts

```diff
@@ -237,32 +237,35 @@ const translations = {
+          'A block explorer is a crucial tool for navigating and understanding blockchain networks. It acts as a user-friendly interface to inspect and analyze the data stored on a blockchain. With a block explorer, users can explore transaction histories, view account balances, and track the progress of individual blocks and transactions.',
+        tip: 'Look for the scriptSig(Hex) category nested in coinbase input after expanding the transaction within the block.',
       },
-      transacting: {
+      genesis_three: {
+        bash_heading: 'Bash Commands',
+        bash_paragraph:
+          'We are using some basic bash command and options to reverse the compression satoshi did on the text put into the genesis block.',
+        tip: "The 'xxd' command turns a file into hex and adding the '-r' tag reverses it allowing hex to be translated into text. The '-p' tag then prints it to the console allowing you to read the output.",
+      },
+      transacting_two: {
         transactions_heading: 'Transactions',
         transactions_paragraph:
           "A transaction in the world of cryptocurrencies is akin to a single atomic payment, where it involves the process of destroying existing coins and creating new ones. When someone initiates a cryptocurrency transaction, they're essentially instructing the blockchain to transfer a specific amount of coins from one digital wallet to another. To achieve this, the transaction consumes the sender's existing coins and generates new ones for the recipient, effectively updating the ownership ledger.",
         bitcoin_script_heading: 'Bitcoin Script',
         bitcoin_script_paragraph:
           'Bitcoin Script is a simple, stack-based programming language used in Bitcoin transactions to define the conditions under which bitcoins can be spent. It consists of various opcodes (short for operation codes) that specify what operations should be performed on the data within the script.',
-        tip_one:
-          'Given the Op_Code: OP_Pushbytes_33 we are looking for a string 33 bytes long (66 characters total) in the OP_Return of the transaction',
-        tip_two:
-          "When decoding the string with the 'xxd' command, remember that we are not using a variable this time, we need to input the entire string",
+        tip: 'Given the Op_Code: OP_Pushbytes_33 we are looking for a string 33 bytes long (66 characters total) in the OP_Return of the transaction',
+      },
+      transacting_three: {
+        secrets_heading: 'Secrets In Bitcoin',
+        secrets_paragraph:
+          'As we saw from the previous challenge, secrets embedded into bitcoin scripts has been a part of its history since the genesis block. Whether used to simply store some information in the blockchain, to pass information through bitcoin script to give some value external to bitcoin, or to pass messages to others, people have been stroing things in the blockchain other than inputs and outputs since the beginning.',
```

Comment:
```suggestion
          'As we saw from the previous challenge, secrets embedded into bitcoin scripts has been a part of its history since the genesis block. Whether used to simply store some information in the blockchain, to pass information through bitcoin script to give some value external to bitcoin, or to pass messages to others, people have been storing things in the blockchain other than inputs and outputs since the beginning.',
```